ui/admin/tests/e2e/tests/target.spec.js

```diff
@@ -4,12 +4,14 @@
  */
 
 /* eslint-disable no-undef */
-const { test, expect } = require('@playwright/test');
-const { exec, execSync } = require('child_process');
-const { checkEnv, authenticatedState } = require('../helpers/general');
+const {test} = require('@playwright/test');
+const {execSync} = require('child_process');
+const {checkEnv, authenticatedState} = require('../helpers/general');
```

Comment:
Non-blocking but I am curious as to why the spaces before and after `test` and the other imports were removed? I thought this was a setting controlled by our linter.

Reply:
Thank you for pointing this out, @cameronperera 👍 
It happened because I committed changes via an IDE Git plugin and it skipped running the hooks. I've ran the hooks and fixed the formatting now 👍